app/routes/notifications.js

```diff
@@ -2,41 +2,64 @@
 
 const express = require('express');
 const router = express.Router(); // eslint-disable-line new-cap
+const helpers = require('../../lib/helpers');
 const mobilecommons = rootRequire('lib/mobilecommons');
 const logger = app.locals.logger;
 const stathat = app.locals.stathat;
 
-router.post('/reminder', (req, res) => {
-  const northstarUserId = req.body.northstarUserId;
-  const campaignId = req.body.campaignId;
-  const reminderType = req.body.reminderType;
+const allowedReminderTypes = ['signup', 'reportback'];
 
-  if (!northstarUserId || !campaignId || !reminderType) {
-    res.json({ error: 'Missing parameters' });
-    return;
+router.post('/reminder', (req, res) => {
```

Comment:
Refs https://github.com/DoSomething/gambit/pull/766#issuecomment-264667391 -- if Quicksilver was inspecting all incoming messages for conversation, we could just post to an endpoint like `send_message` and send along the CampaignBot message type as a parameter in our POST, instead of having different routes for each Bot message:
e.g. 
* POST `send_message` mobile=2125551234 message_type='reminder' campaign=362
* POST `send_message` mobile=2125557788 message_type='ask_quantity' campaign=2070